### DISCUSSION TITLE
Radio buttons

### DISCUSSION THREAD

**Discussion by mm2270** [IBM/mac-ibm-notifications#58] [Ideas]
It would be cool to see an option for an accessory view type for "radio" or "radiobutton" which would give you something along these lines.
<img width="412" alt="Screen Shot 2021-11-05 at 11 59 27 AM" src="https://user-images.githubusercontent.com/3653600/140544166-e56ada46-7028-4965-8c90-9b45a94b9036.png">
While this isn't the most common type of selection option in dialogs, I could see some use for it. Radio buttons automatically create the option to only select a single choice, unlike with checkboxes where 2 or more items can end up selected.

The screenshot above is from the classic cocoaDialog utility by the way, which IBM Notifier is a very worthy replacement for.

**Comment by SMartorelli** [comment]
Hi, we can add an optional parameter (such as "/single_selection") to configure our "checklist" accessory view in order to fit the behavior you just described. How does this sounds for you?

**Reply by mm2270** [reply]
Actually sure, that could work.

Of course, after I submitted this idea, I realized that in some ways the drop down menu AV sort of fits this description since you can only select one item from a drop down. But radio buttons/check boxes can be optimal in a dialog when there is only 2 choices to make and only one can be selected. If you can find a way to make a checkbox AV only accept one selection, that would definitely suffice.

**Reply by SMartorelli** [reply]
Ok, I'll anyway create a FR to develop the solution to enable the single selection on the checklist accessory view.
